### DISCUSSION TITLE
Is it safe to clear `cacheDir` once in a while?

### DISCUSSION THREAD

**Discussion by xfzv** [twpayne/chezmoi#2949] [Q&A] [answered]
I noticed that my `cacheDir` (`$XDG_CACHE_HOME/chezmoi`) is now 10GB.

Is it safe to clear it once in a while?

**Comment by twpayne** [comment, marked as answer]
Yes, it only stores downloaded external archives and files for now. It is safe to delete.

**Reply by xfzv** [reply]
I see, thank you.
